hack/README.md

```diff
@@ -69,6 +70,21 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `VERBOSITY`: (default: 1) Set global log level verbosity
 - `PULL_POLICY`: (default: IfNotPresent) Set global CDI pull policy
 
+#### Execute Functional Tests
+To Run Tests
+```
```

Comment:
We need to document the KUBEVIRT_PROVIDER variable and add a link to the available options.

Maybe https://github.com/kubevirt/kubevirtci/blob/master/README.md ?

Reply:
Well we specifically opted to support just k8s-1.10.4 and os-3.10.0 right now, to keep the logic that determines which image to grab simple, with the provision that if k8s-1.11.x or os-3.11.x comes out that we will switch to those. So those two are the only actual options available right now.